### PR TITLE
Mod 3 Lab 3 

### PR DESCRIPTION
**Pull request by DanielMartinez5**
updates to image and a little grammer

### CODE REVIEW

File: Module 3 Lab/QGIS 2.6/Module 3 Lab.md [hunk 1]
```diff
@@ -0,0 +1,452 @@
...
+
+In this task, we will look at a map of the contiguous United States using a few different projections. 
+
+1.	In QGIS Desktop, open the project, Country View.qgs (Figure below) by clicking Project -> Open.
+
+![Country View](figures/Country_View.png "Country View")
+
+Country View.qgs is an “unprojected” map of the lower 48 states comprised of three themes: United States, LA & NYC, and Circle (centered on -97.50, 39.00).  You can see it does not look quite right given the default projection.  The circle shows some obvious skewing.  
+
+The distance property is more difficult to judge.  The known distance between Los Angeles and New York is approximately 3************,962 kilometers.  Let’s see how the “unprojected” map controls distance distortion.
```
Comment (RickSmith) [inline Module 3 Lab/QGIS 2.6/Module 3 Lab.md]: Not sure what the ************** signify. Can you clarify?

Comment (DanielMartinez5) [inline Module 3 Lab/QGIS 2.6/Module 3 Lab.md]: Thats weird, i don't recall putting that in, it should be 3,964 roughly.

Comment (RickSmith) [inline Module 3 Lab/QGIS 2.6/Module 3 Lab.md]: Ok, I will change after merging.

**Comment by DanielMartinez5** [comment]
Ok .

**Pull request merged**